os/arch/arm/src/armv8-m/exc_return.h

```diff
@@ -93,9 +92,12 @@
  */
 
 #define EXC_RETURN_BASE          0xffffffa0
-#else
-#define EXC_RETURN_BASE          0xffffffe1
```

Comment:
Are these changes back ported from Nuttx ?

Reply:
If Yes, in that case, we need to modify the EXC_RETURN_BASE accordingly ..

Reply:
I have checked nuttx, but those values are not working well for us. I have verified these values and they are fine.